Example/Example.xcodeproj/project.pbxproj

```diff
@@ -458,7 +460,9 @@
 				DEVELOPMENT_TEAM = "";
 				ENABLE_PREVIEWS = YES;
 				GENERATE_INFOPLIST_FILE = YES;
+				INFOPLIST_FILE = "Example-Info.plist";
```

Comment:
Can be removed once you remove the `.plist` file

Reply:
Removed in [c7d76ef](https://github.com/StanfordBDHG/ResearchKitOnFHIR/pull/59/commits/c7d76efcf6d5367a838f0df9a6559f719d71bf9a)